src/coreclr/vm/jitinterface.cpp

```diff
@@ -12818,8 +12818,6 @@ PCODE UnsafeJitFunction(PrepareCodeConfig* config,
             "Jitted Entry at" FMT_ADDR "method %s::%s %s\n", DBG_ADDR(nativeEntry),
              ftn->m_pszDebugClassName, ftn->m_pszDebugMethodName, ftn->m_pszDebugMethodSignature));
 
-#if defined(FEATURE_CORESYSTEM)
-
 #ifdef _DEBUG
```

Comment:
It does not look right to call `GetFullyQualifiedNameInfo` in release builds and then do nothing with it. This whole block should be either deleted or placed under `#if 0`.